test/utils/format.js

```diff
@@ -150,12 +152,34 @@ test('collectionHrefFromNameVersion returns a nullValue collection if the collec
   t.is('--', collectionHrefFromNameVersion());
 });
 
-test('fullDate returns the properly formatted date', function (t) {
+test('fullDate and dateOnly returns the properly formatted date', function (t) {
   const date = '2025-04-13T01:15:32.567';
-  const date2 = '2025-04-13T04:15:32.567';
+  const date2 = '2025-01-13T04:15:32.567';
+
   const formattedDate = fullDate(date);
   const formattedDate2 = fullDate(date2);
 
   t.is(formattedDate, '01:15:32 04/13/25');
-  t.is(formattedDate2, '04:15:32 04/13/25');
+  t.is(formattedDate2, '04:15:32 01/13/25'); 
+
+  // test with UTC timezoneFormat using timestamp 
+  const timezoneFormatValue = config.initialTimezoneFormat;
+
+  const fullDateTimestamp = 1744833161789;
+  const fullDateTimestamp2 = 1759434229834;
+  const dateOnlyTimestamp = 1744783161789;
```

Comment:
could you add a case where its like EST or PST or something, basically something other than UTC 👀